aws/lambda/lambda_function.py

```diff
@@ -29,8 +29,8 @@ def handler(event, context):
     logging.debug(f"Article authors: {article.authors}")
     
     # Send error response if the given URL does not contain a valid article
-    # https://github.com/codelucas/newspaper/blob/master/newspaper/article.py#L322
-    if not article.is_valid_body():
+    # https://github.com/codelucas/newspaper/blob/master/newspaper/urls.py#L102
+    if not article.is_valid_url():
```

Comment:
Just curious, does this work for Medium articles as well? 
Many medium article links are formatted `/@username/title-of-the-article`, so was curious if the date check will still work.

Reply:
Some Medium articles like [this](https://levelup.gitconnected.com/how-i-got-in-to-amazon-microsoft-google-all-from-studying-these-resources-31724508ce0e) passes `is_valid_url`.

Some fail, however, like [this one](https://medium.com/double-pointer/guide-to-bloom-filters-50475843dfa2).

I'm now browsing other ways to validate articles...